assets/js/base/components/cart-checkout/shipping-location/index.tsx

```diff
@@ -2,56 +2,21 @@
  * External dependencies
  */
 import { __, sprintf } from '@wordpress/i18n';
-import { ShippingAddress, getSetting } from '@woocommerce/settings';
-import { decodeEntities } from '@wordpress/html-entities';
 
 interface ShippingLocationProps {
-	address: ShippingAddress;
+	formattedLocation: string | null;
 }
 
 /**
  * Shows a formatted shipping location.
  *
- * @param {Object} props         Incoming props for the component.
- * @param {Object} props.address Incoming address information.
+ * @param {Object} props                   Incoming props for the component.
+ * @param {string} props.formattedLocation Formatted location.
```

Comment:
You could move this comment to the interface above and remove the docblock comments here.